scripts/node.sh

```diff
@@ -317,6 +318,8 @@ do
    R) TRACEFILE="${OPTARG}";;
    l) broadcast_invalid_tx=false;;
    L) log_level="${OPTARG}";;
+
+   M) msg "WARNING: deprecated flag -M: Multi BLS is always enabled, and it's safe to remove this flag.";;
```

Comment:
let's not remove this flag, as our internal nodes are using -M by default.

Reply:
I definitely agree with you. So -M flag is not removed but only deprecated. The node.sh will just run as it supposed to.